lib/sacastats_web/controllers/character_controller.ex

```diff
@@ -100,6 +100,26 @@ defmodule SacaStatsWeb.CharacterController do
     ]
   end
 
+  defp build_assigns(info, status, "general") do
+    faction = SacaStats.Utils.maybe_to_int(info["faction_id"])
+    head = SacaStats.Utils.maybe_to_int(info["head_id"])
+
+    ethnicity = get_character_ethnicity(faction, head)
+    sex = get_character_sex(faction, head)
+
+    character_characteristics =
+      ["ethnicity", ethnicity, "sex", sex]
+      |> Enum.chunk_every(2)
+      |> Map.new(fn [k, v] -> {k, v} end)
```

Comment:
Since there's only 2 entries, I think creating a map explicitly is more readable.
```suggestion
    character_characteristics = %{
      "ethnicity" => ethnicity,
      "sex" => sex,
    }
```